scripts/inspect.sh

```diff
@@ -93,19 +137,34 @@ function suggest_fixes {
 
   if iptables -L | grep FORWARD | grep DROP &> /dev/null
   then
-      printf -- '\033[0;33m WARNING: \033[0m IPtables FORWARD policy is DROP. '
-      printf -- 'Consider enabling traffic forwarding with: sudo iptables -P FORWARD ACCEPT \n'
-      printf -- 'The change can be made persistent with: sudo apt-get install iptables-persistent\n'
+    printf -- '\033[0;33m WARNING: \033[0m IPtables FORWARD policy is DROP. '
+    printf -- 'Consider enabling traffic forwarding with: sudo iptables -P FORWARD ACCEPT \n'
+    printf -- 'The change can be made persistent with: sudo apt-get install iptables-persistent\n'
   fi
 
   ufw=$(ufw status)
   if echo $ufw | grep "Status: active" &> /dev/null && ! echo $ufw | grep cbr0 &> /dev/null
   then
-      printf -- '\033[0;33m WARNING: \033[0m Firewall is enabled. Consider allowing pod traffic '
-      printf -- 'with: sudo ufw allow in on cbr0 && sudo ufw allow out on cbr0\n'
+    printf -- '\033[0;33m WARNING: \033[0m Firewall is enabled. Consider allowing pod traffic '
+    printf -- 'with: sudo ufw allow in on cbr0 && sudo ufw allow out on cbr0\n'
   fi
-}
 
+  # check for selinux. if enabled, print warning.
+  if getenforce 2>&1 | grep 'Enabled' > /dev/null
+  then
+    printf -- '\033[0;33m WARNING: \033[0m SElinux is enabled. Consider disabling it.\n'
+  fi
+
+  # check for docker
+  if [ -d "/etc/docker/" ] && ! sudo grep localhost:32000 /etc/docker/daemon.json > /dev/null; then 
```

Comment:
This gave me the following ugly output:
```
grep: /etc/docker/daemon.json: No such file or directory
 WARNING:  Docker is installed. You should mark the registry as insecure. Add the following to /etc/docker/deamon.json : 
 {
         "insecure-registries" : ["localhost:32000"] 
 }
```
Please test for the existence of the file with `[ -f "/etc/docker/daemon.json" ]`